tensorboard/plugins/core/core_plugin_test.py

```diff
@@ -53,6 +69,35 @@ def testRoutesProvided(self):
     self.assertIsInstance(routes['/data/logdir'], collections.Callable)
     self.assertIsInstance(routes['/data/runs'], collections.Callable)
 
+  def testFlag(self):
+    loader = core_plugin.CorePluginLoader()
+    loader.fix_flags(FakeFlags(inspect=True, logdir='/tmp'))
+    loader.fix_flags(FakeFlags(inspect=True, event_file='/tmp/event.out'))
+    loader.fix_flags(FakeFlags(inspect=False, logdir='/tmp'))
+    loader.fix_flags(FakeFlags(inspect=False, db='sqlite:foo'))
+    # User can pass both, although the behavior is not clearly defined.
```

Comment:
FWIW, as implemented the behavior is that it will use the DB for all summary data, but fall back to the provided logdir for non-summary data (e.g. the projector).

Reply:
Yup but I meant to say that it is not well documented.